PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/sparkserverless/serverexplore/ui/SparkServerlessProvisionDialog.java

```diff
@@ -121,6 +123,14 @@ public void focusLost(FocusEvent e) {
         // TODO: will be removed in the final version
         this.adlAccountField.setText(adlAccountNode.getAdlAccount().getName());
         this.storageRootPathLabel.setText(Optional.of(account.getStorageRootPath()).orElse(""));
+        this.getWindow().addWindowListener(new WindowAdapter() {
+            @Override
+            public void windowActivated(WindowEvent e) {
```

Comment:
Refer to https://docs.oracle.com/javase/tutorial/uiswing/events/windowlistener.html, `windowOpened` may be more proper.